spec/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
 require "codeclimate-test-reporter"
 require "coveralls"
-# require 'simplecov'
-Coveralls.wear!
-# SimpleCov.start
+ require 'simplecov'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.